packages/hub/src/app/users/[username]/models/[slug]/relative-values/[variableName]/layout.tsx

```diff
@@ -53,6 +63,8 @@ export default function RelativeValuesModelLayout({
     throw new Error("Not found");
   }
 
+  const def = revision.forRelativeValues.definition;
```

Comment:
it's awkward that we use the word "definition" on line 68, though it means something different. We could use better terminology.